test-integ/topoutil/asserter_blankspace.go

```diff
@@ -0,0 +1,261 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package topoutil
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/consul/sdk/testutil/retry"
+	"github.com/hashicorp/consul/testing/deployer/topology"
+	"github.com/stretchr/testify/require"
+)
+
+// CheckBlankspaceNameViaHTTP calls a copy of blankspace and asserts it arrived
+// on the correct instance using HTTP1 or HTTP2.
+func (a *Asserter) CheckBlankspaceNameViaHTTP(
+	t *testing.T,
+	service *topology.Service,
+	upstream *topology.Upstream,
+	useHTTP2 bool,
+	path string,
+	clusterName string,
+	sid topology.ServiceID,
+) {
+	t.Helper()
+
+	a.checkBlankspaceNameViaHTTPWithCallback(t, service, upstream, useHTTP2, path, 1, func(r *retry.R, remoteName string) {
+		require.Equal(r, fmt.Sprintf("%s::%s", clusterName, sid.String()), remoteName)
+	}, func(r *retry.R) {})
+}
+
+// CheckBlankspaceNameTrafficSplitViaHTTP is like CheckBlankspaceNameViaHTTP
+// but it is verifying a relative traffic split.
+func (a *Asserter) CheckBlankspaceNameTrafficSplitViaHTTP(
+	t *testing.T,
+	service *topology.Service,
+	upstream *topology.Upstream,
+	useHTTP2 bool,
+	path string,
+	expect map[string]int,
+	epsilon int,
+) {
+	got := make(map[string]int)
```

Comment:
~ Should this and `CheckBlankspaceNameTrafficSplitViaTCP` also call `t.Helper()`?

Reply:
Yes I had missed 3.